README.md

```diff
@@ -3,3 +3,14 @@
 </h1>
 
 An opinionated Kubernetes distribution with a focus on long-term on-prem deployments
+
+## How to run tests
```

Comment:
Rename to End-to-End Testing

Reply:
Add a `Requirements` section - just python3.4+, tox and vagrant with virtualbox provider.